apis/management.cattle.io/v3/cluster_types.go

```diff
@@ -74,6 +74,7 @@ type ClusterSpec struct {
 	DefaultClusterRoleForProjectMembers  string                               `json:"defaultClusterRoleForProjectMembers,omitempty" norman:"type=reference[roleTemplate]"`
 	DockerRootDir                        string                               `json:"dockerRootDir,omitempty" norman:"default=/var/lib/docker"`
 	EnableNetworkPolicy                  *bool                                `json:"enableNetworkPolicy" norman:"default=false"`
+	RotateCertificates                   *RotateCertificateInput              `json:"rotateCertificates,omitempty"`
```

Comment:
should rotateCertificates be a part of rkeConfig? as it's RKE clusters feature only